content/community/gallery.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Community Gallery"
+weight: 1
+gallery_section:
+  enable: true
+  title: "Data Conversion"
+  content: "Tools and resources for converting data to NWB format"
+  boxes:
+    - title: "NeuroConv Catalog"
+      content: "Real-world examples of labs using NeuroConv to convert their data to NWB files. Each project listed contains a description and a link to an open GitHub repository. Many of the projects listed use advanced customization features beyond what is demonstrated in the core NeuroConv conversion gallery."
+      image: "/images/community_gallery/neuroconv_gallery.png"
+      url: "https://neuroconv.readthedocs.io/en/main/catalogue/index.html"
```

Comment:
The URL gives a 404, I think this may be correct link

```suggestion
      url: "https://neuroconv.readthedocs.io/en/stable/conversion_examples_gallery/index.html"
```